src/client/pages/OfferNew/index.tsx

```diff
@@ -157,6 +157,9 @@ export const OfferNew: React.FC = () => {
 
   const handleCheckoutToggle = (open: boolean) => {
     toggleCheckout(open)
+    if (offerData) {
+      trackOfferGTM(EventName.CheckoutOpen, offerData, false)
+    }
```

Comment:
This does not seem to belong to this PR. Was this mistakenly deleted before?

Reply:
It also helps in this PR specifically because you can track the number of people that actually open the checkout  compared to the number of people who actually sign / who put in their phone number.